models/user.go

```diff
@@ -60,8 +83,8 @@ func (u *User) Map(data []byte) error {
 	if err := u.Validate(); err != nil {
 		h.L.WithFields(logrus.Fields{
 			"input": string(data),
-		}).Error("Invalid input")
-		return NewError(InvalidInputCode, "Invalid input")
+		}).Error(err.Error())
```

Comment:
You probably need to update the integration tests to manage the new error message